doc/sphinx/source/getting_started/outputdata.inc

```diff
@@ -4,19 +4,155 @@
 Output
 ******
 
+ESMValTool automatically generates a new output directory with every run. The location is determined by 
+your config.yml's entry: output_dir variable, the recipe name, and 
+the date and time, using the the format: YYYYMMDD & HHMMSS.
+
+For instance, a typical output location would be: 
+output_directory/recipe_ocean_amoc_20190118_1027/
+
+This is effectively produced by the combination:
+$output_dir/recipe_name_YYYYMMDD_HHMMSS/
+
+
+
+This directory will contain 4 further directorys:
+
+1. preproc: This directory contains all the preprocessed netcdfs data & metadata.
+
+2. run: This directory includes all log files, a copy of the recipe, settings.yml, metadata.yml (more on these below). 
```

Comment:
"and the interface files settings.yml, metadata.yml".

Reply:
```suggestion
2. run: This directory includes all log files, a copy of the recipe, a summary of the resource usage, and the `settings.yml`_ and `metadata.yml`_ interface files. 
```